service/src/main/java/com/softserve/borda/services/impl/BoardColumnServiceImpl.java

```diff
@@ -52,31 +49,14 @@ public boolean deleteBoardColumnById(Long id) {
 
     @Override
     public List<BoardColumn> getAllBoardColumnsByBoardId(Long boardId) {
-        return boardService.getBoardById(boardId).getBoardColumns();
+        return boardColumnRepository.getAllBoardColumnsByBoardId(boardId);
     }
 
     @Override
-    public BoardColumn addBoardColumnToBoard(Long boardId, Long boardColumnId) {
-        Board board = boardService.getBoardById(boardId);
+    public BoardColumn moveBoardColumnToBoard(Long boardId, Long boardColumnId) {
         BoardColumn boardColumn = getBoardColumnById(boardColumnId);
-        board.getBoardColumns().add(boardColumn);
-        boardService.update(board);
-        return boardColumn;
-    }
-
-    @Override
-    public boolean deleteBoardColumnFromBoard(Long boardId, Long boardColumnId) {
-        try {
-            Board board = boardService.getBoardById(boardId);
-            BoardColumn boardColumn = getBoardColumnById(boardColumnId);
-            board.getBoardColumns().remove(boardColumn);
-            deleteBoardColumnById(boardColumn.getId());
-            boardService.update(board);
-            return true;
-        } catch (Exception e) {
-            log.severe(e.getMessage());
-            return false;
-        }
+        boardColumn.setBoardId(boardId);
+        return update(boardColumn);
```

Comment:
It can be much easier, any ideas how we can refactor update in this case?
Hint: you can avoid update of all attributes of a board column and even avoid select of  board column